TrkDiag/inc/ComboHitInfo.hh

```diff
@@ -4,6 +4,17 @@
 namespace mu2e {
   // info about each hit in the combo hit
   struct ComboHitInfo {
+    // my addition vvvvvvvvvv
+    Float_t _posx, _posy, _posz; // position of this hit
```

Comment:
Please just embed an XYZVec instead of separate floats for positions, etc.

Reply:
I will remove these additions now